test/internal/helpers/http.go

```diff
@@ -21,6 +22,15 @@ func DefaultHTTPClient() *http.Client {
 	}
 }
 
+// RetryableHTTPClient returns a client that should be used when calling external services that might temporarily fail
+// (e.g. Konnect APIs), and we don't want them to affect the test results.
+func RetryableHTTPClient() *http.Client {
```

Comment:
How about making a wrapper function to return a retryable client based on the input `http.Client`?

Reply:
👍 PTAL